interview-questions.md

```diff
@@ -7,55 +7,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer:git is a open sourced version control system used by programmers.
+  git is the tool used to track and manage your code source history and github is the cloud base hosting service that lets you manage git repositories.
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: true and false
 
-  Researched answer:
+  Researched answer: a binary variable, having two possible values called “true” and “false.”
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: Index is the address of a given character and value is the character in the address
 
   Researched answer:
-
+  An index is a numerical representation of an item's position in a sequence. This sequence can refer to many things: a list, a string of characters, or any arbitrary sequence of values. In some programming languages, an array can be defined by key-value pairs, where a specific key points to a value.
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer:pre programed Functions and Objects in an application
 
-  Researched answer:
+  Researched answer:A function that is built into an application and can be accessed by end-users.
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: Pomodoro is a pair programming technique where a timer is set to a period of time to switch and split the work between to programmers as one navagates and the other drives.
 
-  Researched answer:
+  Researched answer:s Pomodoro a pairing technique that forces developers to split their work time into smaller units. The units of work consist of 25-minute-long sessions with 5-minute breaks in between, and a longer 20-minute break after every 4 sessions.
```

Comment:
https://medium.com/@stanislav.hamara

Reply:
Good Pair Programing techniques allow two programmers to share and responsiblities of the coding process by switching in a timely manner the roles of navigating, which consits of constantly looking over the code for errors and reserching while the  driver inputs the code.